content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -37,7 +47,65 @@ In the `generator` block of your `schema.prisma` file, add the following:
 binaryTargets = ["native", "linux-arm64-openssl-1.0.x"]
 ```
 
-## Loading environment variables via a <inlinecode>.env</inlinecode> file
+### Prisma CLI binary targets
+
+While we do not recommend running migrations in an AWS Lambda context, some applications will require it. In these cases, you can use the [PRISMA_CLI_BINARY_TARGETS](https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_cli_binary_targets) environment variable to make sure that `prisma migrate` has access to the correct schema engine.
+
+In the case of AWS lambda, you will want to have the following line in your `.env` file
+
+```env file=.env
+PRISMA_CLI_BINARY_TARGETS=native,rhel-openssl-1.0.x
+```
+
+<Admonition>
+
+`prisma migrate` is a command in the `prisma` package. Normally, this package is installed as a dev dependency. Depending on your setup, you may need to install this package as a dependency instead in order for the command to be available in AWS Lambda
+
+</Admonition>
+
+### Connection pooling
+
+Generally, when you use a FaaS (Function as a Service) environment to interact with a database, every function invocation can result in a new connection to the database. This is not a problem with a constantly running Node.js server. Therefore, it is beneficial to pool DB connections to get better performance. You can use [Accelerate](/data-platform/accelerate) to solve this issue. For other solutions, see the [connection management guide for serverless environments](/guides/performance-and-optimization/connection-management#serverless-environments-faas).
+
+## AWS SAM guidance
+
+This section covers resolving issues when using the AWS SAM framework. Check the other sections for notes on [AWS Lambda in general](#general-aws-lambda-guidance), [Serverless Framework](#serverless-framework-guidance), or [SST](#sst-guidance).
```

Comment:
I think the links to the other sections are a bit off - we have the TOC on the right etc that should give enough context.

Reply:
agree, don't think we need to repeat the links at the top of each framework-specific section

Reply:
```suggestion
This section covers resolving issues when using the AWS SAM framework. Please check our other sections for information on deploying without a framework, with SST, or with the Serverless Framework.
```

Reply:
I think that would include not mentioning the other sections again.

Reply:
Ah, I see what you mean. Okay, I'll remove references to the other sections